migrations/20161104170802-sample-remove-name-index.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface, Sequelize) {
+    queryInterface.removeIndex('Samples', 'SampleName');
+  },
+
+  down: function (queryInterface, Sequelize) {
+    queryInterface.addIndex(
+      'Samples',
+      ['name'],
+      {
```

Comment:
I think it would hurt us to have extra indexes. 
See: http://stackoverflow.com/questions/20707182/disadvantages-of-creating-multiple-indexes-in-a-postgresql-table
> indexes have a performance penalty when it comes to INSERTing new rows, DELETEing old ones or UPDATEing existing values of the indexed column, as now the DML statement need not only modify the table's data, but the index's one too